src/NHibernate.Test/NHSpecificTest/EmbIdBidirectional/Fixture.cs

```diff
@@ -0,0 +1,46 @@
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.EmbIdBidirectional
+{
+	public class Fixture : BugTestCase
+	{
+		[Test]
+		[KnownBug("Regression - worked in 5.1.x. Now also Teardown fails because entity cannot be loaded....")]
```

Comment:
`.CreateQuery("delete from BiEmbIdRefIngEntity").ExecuteUpdate()` will not load the entities.